docs/getting_started/tutorial.md

```diff
@@ -12,7 +12,7 @@ If you want to add Wagtail to an existing Django project instead, see [](integra
 
 ### Install dependencies
 
-Wagtail supports Python 3.8, 3.9, 3.10, and 3.11.
+Wagtail supports Python 3.8, 3.9, 3.10, 3.11, and 3.12.
```

Comment:
```suggestion
View the [compatible versions of Python](compatible_django_python_versions) that Wagtail supports.
```

Just thinking, rather than keep this manually in sync we could just reference the docs page that lists this.